src/main/java/com/sintern/domain/Domain.java

```diff
@@ -15,6 +16,7 @@
 @FieldDefaults(level = AccessLevel.PRIVATE)
 public class Domain {
 
+    @JsonIgnore
```

Comment:
Why is this ignored?

Reply:
Because I consulted with someone from FE and they said that they don't need all that information so it would save some resources.